### ISSUE TITLE
Security contacts for node-api not specified or out of date

### ISSUE THREAD

**Issue by joelsmith** [kubernetes/kubernetes#92102]
The purpose of the `SECURITY_CONTACTS` file for each Kubernetes repository is to provide a list of people who can assist the Kubernetes [Product Security Committe](https://github.com/kubernetes/community/tree/master/committee-product-security) in the event that a security issue related to the repository is discovered or disclosed.  As described in the file, those on the list should agree to our [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy).

Please update the [`/staging/src/k8s.io/node-api/SECURITY_CONTACTS`](/kubernetes/kubernetes/blob/master/staging/src/k8s.io/node-api/SECURITY_CONTACTS) file for the node-api repository. After finding people who are willing to work in this capacity, you should add them to the list, then remove PSC members (except any PSC member will be working as a security contact for this repository). The list is GitHub usernames, optionally followed by an email address. If no email address is listed, the PSC will use the email address found on git commits made by the listed user.

The file may already have people listed who are secuirty contacts. In that case, simply remove any PSC members who aren't also security contacts for the repo.

See https://github.com/kubernetes/security/issues/92 for more information

/area security
/committee product-security
/kind cleanup
/lifecycle frozen
/priority important-soon

**Comment by tallclair** [comment]
@nikhita do you know what the process for deleting a repo is? We deleted the staging source for node-api in https://github.com/kubernetes/kubernetes/pull/87503.

/cc @liggitt

**Comment by liggitt** [comment]
> @nikhita do you know what the process for deleting a repo is? We deleted the staging source for node-api in #87503.

we should definitely not delete the repo, it is still referenced from historical versions. we can make a PR directly against master of that repo

**Comment by tallclair** [comment]
Got it. Do you think we'd want to delete it at some point, or should we plan to keep it indefinitely? Would it make sense to archive the repo? Either way, we should probably add a deprecation / EOL notice (I can take that).

**Comment by liggitt** [comment]
> Do you think we'd want to delete it at some point, or should we plan to keep it indefinitely?

deleting code referenced by shipped releases is extremely poor practice. archiving and updating the readme to indicate EOL seems good.

**Comment by nikhita** [comment]
> archiving and updating the readme to indicate EOL seems good.

The repo was removed after v1.17. We have publishing bot rules that will still sync the `release-1.16` and `release-1.17` branches for `node-api` so we can't archive the repo until v1.17 is out of support.

https://github.com/kubernetes/kubernetes/blob/55476fb8723be7eef19e0db3a5a084aded3178eb/staging/publishing/rules.yaml#L1473-L1503

I have created https://github.com/kubernetes/node-api/pull/4 to add an EOL note to the README.
